src/libraries/Common/src/System/Obsoletions.cs

```diff
@@ -59,5 +59,11 @@ internal static class Obsoletions
 
         internal const string GetContextInfoMessage = "Use the Graphics.GetContextInfo overloads that accept arguments for better performance and fewer allocations.";
         internal const string GetContextInfoDiagId = "SYSLIB0016";
+
```

Comment:
Note that there's another open PR that adds `SYSLIB0017` and `SYSLIB0018`, which is why I skipped those.